src/kleinanzeigen_bot/utils/web_scraping_mixin.py

```
@@ -848,22 +848,78 @@ async def web_select(self, selector_type:By, selector_value:str, selected_value:
+                    }}
                 }}
-              }}
-              throw new Error("Option with value {selected_value} not found.");
+
+                throw new Error("Option not found by value or displayed text: " + wanted);
             }}
         """)
         await self.web_sleep()
         return elem
 
+    async def web_select_combobox(self, selector_type:By, selector_value:str, selected_value:Any, timeout:int | float = 5) -> Element:
```

Ruff (0.14.4)

Dynamically typed expressions (typing.Any) are disallowed in `selected_value`

(ANN401)